src/Razor/src/Microsoft.AspNetCore.Razor.Tools/runtimeconfig.template.json

```diff
@@ -0,0 +1,3 @@
+{
+    "rollForwardOnNoCandidateFx": 2
```

Comment:
2 represents "true" I take it?

Reply:
No, it means "Roll forward on [major], [minor] and [patch]" instead of the default "Roll forward on [minor] and [patch]". See [docs here](https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md#runtimeoptions-section-runtimeconfigjson) and [here](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md#roll-forward-in-absence-of-candidate-fx).